js/turtledefs.js

```diff
@@ -97,10 +97,10 @@ function createHelpContent() {
         [_('Expand/collapse collapsable blocks'), _('Expand or collapse start and action stacks.'), 'header-icons/collapse-blocks-button.svg'],
         [_('Home'), _('Return all blocks to the center of the screen.'), 'header-icons/go-home-button.svg'],
         [_('Help'), _('Show these messages.'), 'header-icons/help-button.svg'],
-        [_('Expand/collapse option toolbar'), _('Click this button to expand or collapse the auxillary toolbar.'), 'header-icons/menu-button.svg'],
+        [_('Expand/collapse option toolbar'), _('Located on your right - Click this button to expand or collapse the auxillary toolbar. '), 'header-icons/menu-button.svg'],
         [_('Load samples from server'), _('This button opens a viewer for loading example projects.'), 'header-icons/planet-button.svg'],
-        [_('Load project from files'), _('You can also load projects from the file system.'), 'header-icons/open-button.svg'],
-        [_('Save project'), _('Save your project to a file.'), 'header-icons/save-button.svg'],
+        [_('Load project from files'), _('You can also load projects from the file system.      Tip: There are several projects on famous songs in examples folder, Use them to get quick feel of Music Blocks.  '), 'header-icons/open-button.svg'],
```

Comment:
I think it is easier to find examples from the planet. these examples would only be on the file system if they cloned the project locally.